nucypher/cli/types.py

```diff
@@ -128,7 +127,8 @@ def convert(self, value, param, ctx):
 WEI = click.IntRange(min=1, clamp=False)  # TODO: Better validation for ether and wei values?
 GWEI = DecimalRange(min=0)
 
-__min_authorization = TToken.from_units(Economics._default_min_authorization).to_tokens()
+
+__min_authorization = TToken(40_000, "T").to_tokens()  # TODO right spot for this?
```

Comment:
What's the use for this anyway? The min authorization is whatever is in the contract, and we don't use the `nucypher` CLI for deployment or staking.

Reply:
It's used in a CLI option as a default value - https://github.com/nucypher/nucypher/blob/development/nucypher/cli/options.py#L65 - which is used as an Ursula parameter for specify the seed node minimum stake.

Reply:
huh, but I'm wondering why is that a CLI parameter. The only thing that should matter is what's in the contract, so I don't see how a different value can be used

Reply:
The CLI parameter allows nodes to specify a higher stake requirement for its seed node teacher than the minimum (trust mitigation I guess... 🤷), but you are right that perhaps the logic can be reworked so that if a value is not specified, the min value is obtained from the contract.